projects/notebooks/testing/config.yaml

```diff
@@ -12,7 +12,7 @@ ci_presets:
     clusters.create.ocp.workers.type: m6i.2xlarge
     clusters.create.ocp.compute.type: m6i.2xlarge
     tests.notebooks.users.sleep_factor: 3.0
-    tests.notebooks.test_flavor: ods-ci
+    tests.notebooks.test_flavor: dashboard-scale-test
```

Comment:
can you revert the changes of this file, and instead extend the existing preset, with something like that:
```
dashboard_scale_test:
  tests.notebooks.test_flavor: dashboard-scale-test
  tests.notebooks.minio_bucket_name: ds-pipeline-dspa
  tests.notebooks.users.count: 5
  tests.notebooks.capture_prom_db: false
  clusters.sutest.compute.dedicated: false
  clusters.driver.compute.dedicated: false

```